app/src/main/java/org/permanent/permanent/ui/composeComponents/CustomDropdown.kt

```diff
@@ -88,11 +125,198 @@ fun CustomDropdown(isTablet: Boolean = false) {
                 modifier = Modifier.size(12.dp)
             )
         }
+
+        if (showBottomSheet) {
+            ModalBottomSheet(
+                onDismissRequest = {
+                    showBottomSheet = false
+                }, sheetState = sheetState
+            ) {
+                // Sheet header
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(bottom = 16.dp, start = 20.dp, end = 20.dp),
+                    verticalAlignment = Alignment.CenterVertically,
+                ) {
+                    Image(
+                        painter = painterResource(id = R.drawable.ic_close_middle_grey),
+                        contentDescription = "Plus",
+                        colorFilter = ColorFilter.tint(Color.Transparent),
+                        modifier = Modifier.size(18.dp),
+                    )
+
+                    Text(
+                        modifier = Modifier.weight(1.0f),
+                        text = stringResource(R.string.archive_type),
+                        textAlign = TextAlign.Center,
+                        color = blue900Color,
+                        fontFamily = boldFont,
+                        fontSize = 16.sp,
+                        lineHeight = 24.sp
+                    )
+
+                    Image(
+                        painter = painterResource(id = R.drawable.ic_close_middle_grey),
+                        contentDescription = "Plus",
+                        colorFilter = ColorFilter.tint(blue900Color),
+                        modifier = Modifier
+                            .size(18.dp)
+                            .clickable {
+                                scope
+                                    .launch { sheetState.hide() }
+                                    .invokeOnCompletion {
+                                        if (!sheetState.isVisible) {
+                                            showBottomSheet = false
+                                        }
+                                    }
+                            },
+                    )
+                }
+                // Sheet content
+                Column(
+                    modifier = Modifier.fillMaxHeight(), verticalArrangement = Arrangement.Top
+                ) {
+                    Divider()
+
+                    StorageMenuItem(
```

Comment:
Maybe it would be better to store the info in an array for each archive type and do a ForEach since right now there is a lot of repeated code.